libr/anal/esil.c

```diff
@@ -1676,14 +1676,21 @@ static int esil_poke_some(RAnalEsil *esil) {
 		if (count) {
 			isregornum (esil, count, &regs);
 			if (regs > 0) {
+#ifdef _MSC_VER
+				ut8 *b = (ut8*) malloc(sizeof(ut8) * bytes);
+#else
 				ut8 b[bytes];
```

Comment:
dont ifdef this. variable sized arrays or alloca shouldnt be used in r2land.

you can also assume that max regsize will be 64 and max bytes will be 8. so you can save that malloc here by just using b[8];